indexer/index.js

```diff
@@ -20,6 +24,17 @@ const startProcessing = async () => {
 
   cron.schedule('* * * * *', async() => {
     console.log('Cron job starting: Sync Hubs + Releases');
+    if(arg[2]=="--heap-stats") {
+      console.log("Memory Diagnostics at " + new Date(Date.now()) + ": ");
```

Comment:
can these console logs go into a function, maybe `runHeapDiagnostics` so that we can keep the cron job clean?

Reply:
Done.